crates/prices/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "iron-prices"
```

Comment:
maybe `iron-exchange-rates` would be a better name now that I think about it. what do you think?

Reply:
sounds good!